### PR TITLE
added lioness dragon impl and test outline

### PR DESCRIPTION
**Pull request by katehoffshutta**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -39,7 +39,7 @@ pt_out.csv
 reg_memb.txt
 tar_memb.txt
 gt_panda*.txt
-
+*~
```
Comment (marouenbg) [inline .gitignore:42]: Hey @katehoffshutta what is this supposed to do?

File: tests/test_lioness_dragon.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import pytest
+import os
+from netZooPy.lioness.lioness_for_dragon import LionessDragon
+import pandas as pd
+import numpy as np
+
+def test_lioness_dragon():
+    print("Start LIONESS-DRAGON test!")
+
+s = LionessDragon(layer1 = "lioness/dragon/toy_data_layer1.tsv", layer2 = "lioness/dragon/toy_data_layer2.tsv", output_file = "lioness/dragon/lioness_dragon_results.csv")
```
Comment (marouenbg) [inline tests/test_lioness_dragon.py:10]: you need to add "tests" to these file paths

File: .gitignore (hunk 1 above)
Comment (katehoffshutta) [reply .gitignore:42]: This ignores my emacs backup files and doesn't need to be there for others. I will remove it

**Comment by violafanfani** [comment]
Is this ready to be merged? Should I wait?

**Comment by marouenbg** [comment]
all good on my side, how about you @katehoffshutta ?

**Comment by katehoffshutta** [comment]
Sorry to miss this mention earlier! Thanks for merging. This is ready for devel; needs work before master. In particular, it outputs a very large file with all the networks right now and I want to set it up with an option to write individual files for each network instead.

**Pull request merged**